> thx, will check the clusterizer performance with this.

Dear David, all, 

unfortunately there appeared some errors during fullCI build:
## sw/BUILD/o2checkcode-latest/log
--
```
========== List of errors found ==========
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5887/0/Detectors/CPV/simulation/src/Digitizer.cxx:80:38: error: statement should be inside braces [readability-braces-around-statements]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5887/0/Detectors/CPV/simulation/src/Digitizer.cxx:150:36: error: statement should be inside braces [readability-braces-around-statements]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5887/0/Detectors/CPV/simulation/src/Digitizer.cxx:161:39: error: statement should be inside braces [readability-braces-around-statements]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/5887/0/Detectors/CPV/workflow/src/RawToDigitConverterSpec.cxx:195:25: error: statement should be inside braces [readability-braces-around-statements]
--

```
Machine think that code is not readable which I can debate. However Those errors didn't show up at my laptop. Would it be possible to explain what are those, how to reproduce them locally and how to get rid of it? I don't want to fix something blindly and wait another half day for tests.

Thank you & regards,
Sergey

--- a/Detectors/CPV/simulation/src/Digitizer.cxx
+++ b/Detectors/CPV/simulation/src/Digitizer.cxx
@@ -77,9 +77,10 @@ void Digitizer::init()
 
   //signal thresolds for digits
   //note that digits are calibrated objects
-  for (int i = 0; i < NCHANNELS; i++)
+  for (int i = 0; i < NCHANNELS; i++) {
     mDigitThresholds[i] = o2::cpv::CPVSimParams::Instance().mZSnSigmas *
                           mPedestals->getPedSigma(i) * mCalibParams->getGain(i);
+  }
 }
 
 //_______________________________________________________________________
@@ -147,8 +148,9 @@ void Digitizer::processHits(const std::vector<Hit>* hits, const std::vector<Digi
 
   //finalize output digits
   for (int i = 0; i < NCHANNELS; i++) {
-    if (!mBadMap->isChannelGood(i)) //bad channel -> skip this digit
-      continue;
+    if (!mBadMap->isChannelGood(i)) {
+      continue; //bad channel -> skip this digit
+    }
     if (mArrayD[i].getAmplitude() > mDigitThresholds[i]) {
       digitsOut.push_back(mArrayD[i]);
     }
@@ -158,7 +160,8 @@ void Digitizer::processHits(const std::vector<Hit>* hits, const std::vector<Digi
 float Digitizer::simulatePedestalNoise(int absId)
 {
   //this function is to simulate pedestal and its noise (ADC counts)
-  if (absId < 0 || absId >= NCHANNELS)
+  if (absId < 0 || absId >= NCHANNELS) {
     return 0.;
+  }
   return gRandom->Gaus(0, mPedestals->getPedSigma(absId) * mCalibParams->getGain(absId));
 }

--- a/Detectors/CPV/workflow/src/RawToDigitConverterSpec.cxx
+++ b/Detectors/CPV/workflow/src/RawToDigitConverterSpec.cxx
@@ -192,8 +192,9 @@ void RawToDigitConverterSpec::run(framework::ProcessingContext& ctx)
           if (mBadMap->isChannelGood(absId)) {
             //we need to subtract pedestal from amplidute and calibrate it
             float amp = mCalibParams->getGain(absId) * (ac.Charge - mPedestals->getPedestal(absId));
-            if (amp > 0)
+            if (amp > 0) {
               currentDigitContainer->emplace_back(absId, amp, -1);
+            }
           }
         } else { //pedestal data, no calibration needed.
           currentDigitContainer->emplace_back(absId, (float)ac.Charge, -1);

